framework/audio/iaudioinsert.h

```diff
@@ -0,0 +1,46 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU_AUDIO_IAUDIOINSERT_H
+#define MU_AUDIO_IAUDIOINSERT_H
+
+namespace mu {
+namespace audio {
+class IAudioInsert
```

Comment:
Can you describe in a few words, please, what is the purpose of this interface?

Reply:
This interface is for creation (and insertion) of audio plugins, like equaliser, compressor, reverb, delay etc. It can be applied for the input channel or output of the mixer.

Reply:
IAudioPlugin?

Reply:
We can also call it AudioNode.
The name AudioInsert is not clear and confusing.